include/emp/prefab/ButtonGroup.hpp

```diff
@@ -0,0 +1,44 @@
+#ifndef EMP_BUTTON_GROUP_HPP
+#define EMP_BUTTON_GROUP_HPP
+
+#include "emp/web/Div.hpp"
+
+namespace emp::prefab {
+  /**
+   * Use a ButtonGroup to place buttons of a similar role into the same
+   * container or to save space by placing buttons without gaps between them.
```

Comment:
In addition to this sentence saying /why/ would would want to do use this class, I would add a sentence or two explaining what it does.